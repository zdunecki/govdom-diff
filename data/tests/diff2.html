<html>
<head>

</head>
<body>
    <div>
        Hello
    </div>
    <div>
        World 2
    </div>
</body>
</html>