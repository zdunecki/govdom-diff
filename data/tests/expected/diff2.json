{"nT":2,"i":0,"n":"","tN":"","a":{},"tC":"","cN":[{"nT":3,"i":0,"n":"","tN":"html","a":{},"tC":"","cN":[{"nT":3,"i":0,"n":"","tN":"head","a":{},"tC":"","cN":[{"nT":1,"i":0,"n":"","tN":"","a":{},"tC":"\n\n","cN":[]}]},{"nT":1,"i":2,"n":"","tN":"","a":{},"tC":"\n","cN":[]},{"nT":3,"i":3,"n":"","tN":"body","a":{},"tC":"","cN":[{"nT":1,"i":3,"n":"","tN":"","a":{},"tC":"\n    ","cN":[]},{"nT":3,"i":4,"n":"","tN":"div","a":{},"tC":"","cN":[{"nT":1,"i":4,"n":"","tN":"","a":{},"tC":"\n        Hello\n    ","cN":[]}]},{"nT":1,"i":6,"n":"","tN":"","a":{},"tC":"\n    ","cN":[]},{"nT":3,"i":7,"n":"","tN":"div","a":{},"tC":"","cN":[{"nT":1,"i":7,"n":"","tN":"","a":{},"tC":"\n        World 2\n    ","cN":[]}]},{"nT":1,"i":9,"n":"","tN":"","a":{},"tC":"\n\n","cN":[]}]}]}]}